Wikipedia/Code/ArticleCollectionViewCell+WMFFeedContentDisplaying.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+
+public extension ArticleCollectionViewCell {
+    public func backgroundColor(for displayType: WMFFeedDisplayType) -> UIColor {
+        return displayType == .relatedPages ? UIColor.wmf_lightGrayCellBackground : UIColor.white
+    }
+    
+    public func isSaveButtonHidden(for displayType: WMFFeedDisplayType) -> Bool {
+        return displayType == .pageWithPreview ? false : true
+    }
+    
+    public func configure(article: WMFArticle, contentGroup: WMFContentGroup, layoutOnly: Bool) {
+        let displayType = contentGroup.displayType()
+        
+        let imageWidthToRequest = imageView.frame.size.width < 300 ? traitCollection.wmf_nearbyThumbnailWidth : traitCollection.wmf_leadImageWidth
```

Comment:
For future reference, can we comment why we chose `300` here, or give it a named var for a context hint?